filetests/isa/x86/legalize-i64.clif

```diff
@@ -0,0 +1,27 @@
+; Test the legalization of i64 instructions on x86_32.
+test legalizer
+target i686 haswell
+
+; regex: V=v\d+
+
+function %iadd(i64, i64) -> i64 {
+ebb0(v1: i64, v2: i64):
+    v10 = iadd v1, v2
+    ; check: v1 = iconcat $(v1_lsb=$V), $(v1_msb=$V)
+    ; check: v2 = iconcat $(v2_lsb=$V), $(v2_msb=$V)
```

Comment:
Actually, only the first line requires to be a check; next ones could be nextln, if the lines actually follow in the output. Can you try this, for all the tests in this file, and see if they still pass, please?

Reply:
Sure!